src/MSALWrapper/AuthFlow/IntegratedWindowsAuthentication.cs

```diff
@@ -72,7 +72,8 @@ protected override async Task<TokenResult> GetTokenInnerAsync()
             }
             catch (MsalClientException ex) when (ex.Message.Contains("WS-Trust endpoint not found"))
             {
-                this.logger.LogWarning($"IWA only works on Corp Net, please turn on VPN.");
+                this.logger.LogWarning($"IWA only works on corporate AD backed network, AzureAuth is trying to use other auth flows.");

```

Comment:
This might not be always true if user has selected only IWA mode for their auth. We shouls also try to be consistent with using modes v/s auth flows. We can something similar to "...AzureAuth will use other modes if available"